ShoppingList/ShoppingList/MainWindowViewModel.cs

```diff
@@ -0,0 +1,67 @@
+﻿using GalaSoft.MvvmLight;
+using GalaSoft.MvvmLight.CommandWpf;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Windows.Input;
+
+namespace ShoppingList
+{
+    public class MainWindowViewModel : ViewModelBase, INotifyPropertyChanged
```

Comment:
`ViewModelBase` implements `INotifyPropertyChanged` so you don't need it here.
```suggestion
    public class MainWindowViewModel : ViewModelBase
```